pkg/admission/customresourcevalidation/customresourcevalidationregistration/cr_validation_registration.go

```diff
@@ -26,6 +26,7 @@ var AllCustomResourceValidators = []string{
 	config.PluginName,
 	scheduler.PluginName,
 	clusterresourcequota.PluginName,
+	securitycontextconstraints.PluginName,
```

Comment:
@deads2k is the ordering on this correct?  What comes first, validation or defaulting?  My guess would be the latter.  And I forget how validating vs mutation are ordered.